config/sync/views.view.recipe_list.yml

```diff
@@ -383,6 +383,33 @@ display:
       tags:
         - 'config:field.storage.node.field_description'
         - 'config:field.storage.node.field_image'
+  block_1:
```

Comment:
I think the merge conflict you resolved resulted in reverting the fixes Gabor did in another PR. I suggest you discuss with him how this file should look like.